server/vulnerabilities/msrc/parsed/security_bulletin.go

```diff
@@ -45,6 +46,9 @@ func UnmarshalBulletin(fPath string) (*SecurityBulletin, error) {
 	if err != nil {
 		return nil, err
 	}
+	for pID, name := range bulletin.Products {
+		bulletin.Products[pID] = NewProductFromFullName(string(name))
+	}
```

Comment:
Utilize the new logic in `NewProductFromFullName` to add version strings when needed.

Reply:
i'm not sure this is needed here as NewProductFromFullName is already modifying the published feed here: https://github.com/fleetdm/fleet/blob/cf6e15f6a4e271a1f52ead2098ad610c24e9fd21/server/vulnerabilities/msrc/parser.go#L50

Reply:
I think I needed it here for when we process existing `.json` files that were saved without the versions added to the product names.